server/services/visitSessionsService.ts

```diff
@@ -191,6 +191,23 @@ export default class VisitSessionsService {
     return visit
   }
 
+  async cancelVisit({
+    username,
+    reference,
+    outcome,
+  }: {
+    username: string
+    reference: string
+    outcome: OutcomeDto
+  }): Promise<Visit> {
+    const token = await this.systemToken(username)
+    const visitSchedulerApiClient = this.visitSchedulerApiClientBuilder(token)
+
+    const visit = await visitSchedulerApiClient.cancelVisit(reference, outcome)
```

Comment:
I think if you do a return, you can also remove the `await` so `return visitSched.....` ? Less lines 😄

Reply:
Good spot! Fixed